### PR TITLE
chore: upgrade tailwindcss to v3

### PR DESCRIPTION
**Pull request by bigint**
Everything works as expected, no ui breaks 😄

As a part of #1452

Inherited from https://github.com/snapshot-labs/snapshot/pull/1251

### CODE REVIEW

File: components.d.ts [hunk 1]
```diff
@@ -71,6 +71,7 @@ declare module 'vue' {
     PluginSafeSnapInputMethodParameter: typeof import('./src/components/Plugin/SafeSnap/Input/MethodParameter.vue')['default']
     PluginSafeSnapModalOptionApproval: typeof import('./src/components/Plugin/SafeSnap/Modal/OptionApproval.vue')['default']
     PluginSafeSnapSafeTransactions: typeof import('./src/components/Plugin/SafeSnap/SafeTransactions.vue')['default']
+    PluginSafeSnapTooltip: typeof import('./src/components/Plugin/SafeSnap/Tooltip.vue')['default']
```
Comment (bigint) [inline components.d.ts:74]:
```suggestion
```

**Comment by bigint** [comment]
cc @samuveth @bonustrack

**Comment by samuveth** [comment]
Looks good! This still needs to be changes https://tailwindcss.com/docs/upgrade-guide#replace-variants-with-layer

**Comment by bigint** [comment]
@samuveth done can you check now 🙇🏻

**Review by samuveth — APPROVED** [review]
👍

**Pull request merged**